File: src/main/java/de/maggiwuerze/xdccloader/events/channel/server/ChannelCreationEvent.java
package de.maggiwuerze.xdccloader.events.channel.server;

import de.maggiwuerze.xdccloader.events.EntityCreationEvent;
import de.maggiwuerze.xdccloader.model.entity.Channel;

public class ChannelCreationEvent extends EntityCreationEvent<Channel> {

    public ChannelCreationEvent(Object source, Channel channel) {
        super(source, channel);
    }

}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/channel/server/ChannelCreationEvent.kt
package de.maggiwuerze.xdccwebloader.events.channel.server

import de.maggiwuerze.xdccwebloader.events.EntityCreationEvent
import de.maggiwuerze.xdccwebloader.model.entity.Channel

class ChannelCreationEvent(source: Any, channel: Channel) : EntityCreationEvent<Channel>(source, channel)